From 47543117336c4bc2168bf3060fc182daf553ecc7 Mon Sep 17 00:00:00 2001
From: Christian Simon <simon@swine.de>
Date: Sat, 27 Mar 2021 18:39:09 +0000
Subject: [PATCH] Ensure zfs device names are gather with full disk path

---
 cloudinit/util.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cloudinit/util.py b/cloudinit/util.py
index 769f3425..dbb97bf6 100644
--- a/cloudinit/util.py
+++ b/cloudinit/util.py
@@ -2211,7 +2211,7 @@ def get_device_info_from_zpool(zpool):
         LOG.debug('Cannot get zpool info, no /dev/zfs')
         return None
     try:
-        (zpoolstatus, err) = subp.subp(['zpool', 'status', zpool])
+        (zpoolstatus, err) = subp.subp(['zpool', 'status', '-P', zpool])
     except subp.ProcessExecutionError as err:
         LOG.warning("Unable to get zpool status of %s: %s", zpool, err)
         return None
-- 
2.29.3

